Sources/ApolloWebSocket/WebSocketTransport.swift

```diff
@@ -270,7 +270,10 @@ public class WebSocketTransport {
   }
 
   func sendHelper<Operation: GraphQLOperation>(operation: Operation, resultHandler: @escaping (_ result: Result<JSONObject, Error>) -> Void) -> String? {
-    let body = requestBodyCreator.requestBody(for: operation, sendOperationIdentifiers: self.sendOperationIdentifiers)
+    let body = requestBodyCreator.requestBody(for: operation,
```

Comment:
do not change this.

Reply:
This has to change due to the removal of the default values for these parameters